packages/react/src/components/Storage/FileUploader/FileUploader/FileUploader.tsx

```diff
@@ -346,13 +346,13 @@ export function FileUploader({
   );
 
   useEffect(() => {
-    if (shouldAutoProceed && autoLoad && !hasMaxFilesError) {
+    if (shouldAutoUpload && autoLoad && !hasMaxFilesError) {
```

Comment:
Do we need `autoLoad`?

Reply:
Nope. Simplified by removing autoload state and calling `handleUploadFile` from `UploadTracker` component instead